p/db/index.js

```diff
@@ -55,20 +55,18 @@ const createParseJourney = (profile, opt, data) => {
 		// 	targetCtx: 'D',
 		// 	buttonText: 'To offer selection'
 		// } ]
-		res.price = {amount: null, hint: 'No pricing information available.'}
```

Comment:
Please assign `null` as the default value.

Reply:
Added `null` as default